packages/compat/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@eslint/compat",
```

Comment:
Since this is a scoped package, do we maybe need to mark it as public like we're doing in `@eslint/create-config`?

https://github.com/eslint/create-config/blob/12886eec4b228bb6a23200a9c2231ba938376006/package.json#L28-L30

Reply:
Ooh yes, good catch!